client/app/controllers/map.js

```diff
@@ -3,6 +3,18 @@ import Ember from 'ember';
 export default Ember.Controller.extend({
   isOnline: true,
 
+  trackPollingOffline: Ember.observer('isOnline', function() {
+    if (!this.get('isOnline')) {
+      this.send('trackEvent', 'polling-offline');
```

Comment:
I'm not sure how this event would ever be tracked – if the user lost their network connection (which is normally the only case `isOnline` would be false), the request to track the event will fail too. Although, even if we reworked this code to somehow queue the event for later, is "the user lost their network connection" really a valuable thing to track? It doesn't seem very actionable; connections drop in and out all the time.